compiler/src/dotty/tools/dotc/typer/Applications.scala

```diff
@@ -1293,7 +1293,19 @@ trait Applications extends Compatibility { self: Typer with Dynamic =>
         val formals1 =
           if (tp1.isVarArgsMethod && tp2.isVarArgsMethod) tp1.paramInfos.map(_.repeatedToSingle)
           else tp1.paramInfos
-        isApplicable(alt2, formals1, WildcardType) ||
+
+        class IsAsSpecific(methRef: TermRef, args: List[Type], resultType: Type)(implicit ctx: Context)
```

Comment:
Can we move that class so that it follows directly class `ApplicableToTypes`?

Reply:
Done.